cli/src/android/add.ts

```diff
@@ -22,9 +22,14 @@ export async function addAndroid(config: Config) {
     },
   );
 
-  await runTask(`Syncing Gradle with ${c.input('./gradlew')}`, async () => {
-    return createLocalProperties(config.android.platformDir);
-  });
+  await runTask(
+    `Syncing Gradle with ${c.input(
+      process.platform === 'win32' ? 'gradlew.bat' : './gradlew',
+    )}`,
```

Comment:
createLocalProperties does more than running gradlew, so if you want to specify what's going on, the new message is still not correct. Anyway, I don't see the point of extending this message and add this complexity to say that it's running gradlew

also, not related to the PR, but it made me realize that we run gradlew in several places, but that won't work on windows

Reply:
Good point. Maybe "Syncing Gradle" isn't the best message? In any case, I'll revert for now.

> we run gradlew in several places, but that won't work on windows

Yeah, I noticed as well. We also aren't using `cross-spawn` which is necessary once subprocess invocations become more complex.